src/Resources/ResourceManager/Implementation/Policy/SetAzurePolicyDefinition.cs

```diff
@@ -85,7 +85,6 @@ public class SetAzurePolicyDefinitionCmdlet : PolicyCmdletBase
         /// Gets or sets the policy definition mode parameter.
         /// </summary>
         [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = PolicyHelpStrings.NewPolicyDefinitionModeHelp)]
-        [PSArgumentCompleter(PolicyDefinitionMode.All, PolicyDefinitionMode.Indexed)]
```

Comment:
You cannot remove the argument completer for this.  Are these values still allowed?  If not, this is a breaking change.  If there are additional known allowed values, please add them.  PSArgumentCompleter just allows the user to tab through known values, but the user can also provide values outside this set.

Reply:
Thanks, will remove argument completer as the current values are still valid.